device/bgp/bgpadvanced.yaml

```diff
@@ -45,3 +45,20 @@ components:
             If not configured, MD5 authentication will not be enabled.
           type: string
           x-field-uid: 5
+        passive_mode:
+          description: |-
+            If enabled then the BGP Finite State Machine will passively wait 
+            for the remote BGP peer to establish the BGP TCP connection. 
+            If the passive mode is true, BGP peer will wait for the remote BGP peer 
+            to issue an Open request before an Open message is sent.
+          type: boolean
+          default: false
+          x-field-uid: 6
+        local_listen_port:
```

Comment:
I think the attribute defined here is expected to do two jobs : 
1) Define the listening port used to accept connections from peer .
2) Define the port used to connect to remote peer when passive_mode is false.
If this is the expectation, the name should probably be changed to listen_port to remove the assumption that it will be used only for local_listen_port and not when connecting to remote peer.
Otherwise , we need two attributes , one for local_listen_port and one for neighbor_port .